### ISSUE TITLE
Activity fails when launching app from notification

### ISSUE THREAD

**Issue by speckledpear** [EddyVerbruggen/nativescript-local-notifications#84]
I am developing an Android application which sends daily notifications. However every time I open the app from the notification while the app is not running in the background it crashes once I navigate to a new page. My error is:

```
TypeError: Cannot read property getWindow of undefined
File: file:///data/data/org.nativescript.RealityKeeper/files/app/commonlogic.js, line: 35, column: 50

StackTrace: 
	Frame: function:setAndroidBackground, file:file:///data/data/org.nativescript.RealityKeeper/files/app/commonlogic.js, line: 35, column: 51
```

The code that causes this is:
```
var window = app.android.startActivity.getWindow();
window.setStatusBarColor(androidColor.android);
window.setNavigationBarColor(androidColor.android);
```

And the notification is scheduled with this:
```
notificationHandler.schedule([{
            id: 1,
            title: 'Reality Keeper',
            body: 'Vergeet niet je vragen in te vullen!',
            interval: 'day',
            at: date
        }]).then(() => {
            console.log("Notification scheduled");
        }, (error) => {
            console.log("ERROR", error);
        });
```
I've set my launch mode to singleTop however this does not help.

It looks like the app doesn't launch properly, is this a bug or am I doing something wrong?
Any help would be appreciated, thank you!

**Comment by EddyVerbruggen** [comment]
This is probably fixed with plugin version 3.0.0 because I've changed how the app "launches" when a notification was pressed.

**Comment by hackerunet** [comment]
Can you provide a implementation sample to launch an application and navigate to certain view or at least launch the app in a module or given view? I'm looking for alternatives to go directly to certain application module after clicking on the notification but I haven't been able to find this, do you have maybe a reference to another source of information were anyone has achieved this before?
thanks
